docs/application/dotnet/guides/app-management/data-control.md

```diff
@@ -32,57 +32,65 @@ The main features of the Tizen.Applications.DataControl namespace include:
 
 ## Prerequisites
 
-The data control use cases involve 2 applications. Each application plays a different role: one as the consumer, the other as the provider.
+The data control use cases involve two applications. Each application plays a different role: one as a consumer, the other as a provider.
+
+To enable your application to use the data control functionality, you have to include [Tizen.Applications.DataControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.DataControl.html) namespace.
+```
+using Tizen.Applications.DataControl;
+```
+
+#### Consumer
+Consumer application needs to request specific permission by adding the following privileges to the `tizen-manifest.xml` file:
+```
+<privileges>
+    <privilege>http://tizen.org/privilege/datasharing</privilege>
+    <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+</privileges>
+```
+
+#### Provider
+
+For your provider, you need to include a few things in `tizen-manifest.xml`. If you are using Visual Studio, double-click `tizen-manifest.xml`, and in the manifest editor, go to **Advanced > Data Control**, and click **Add** to add the provider details. Add the `Read` and `Write` access rights to both `SQL` and `Map` types, as needed.
+
+You can set the data access to be trusted, allowing other applications signed with the same certificate to access the data. You can also define privileges to restrict access to applications having the specific privileges.
+
+If you are not using Visual Studio, this is sample `tizen-manifest.xml`, that you can modify for your needs accordingly.
```

Comment:
@MathewP1, since there is a code following you need add a colon at the end of the sentence. Please make this change.

If you are not using Visual Studio, the following is the tizen-manifest.xml sample that you can modify according to your requirement: